Page for users' guide on pedestal estimation tool

tibaldo opened the pull request:
I wrote a first minimalistic version of a documentation page for the users' guide on the pedestal estimation tool. How does it look? Is there anything that should be added or changed?

jlenain commented:
Hi @tibaldo!
Thanks a lot for this! The documentation builds nicely locally:
![image](https://github.com/user-attachments/assets/6ee9adb2-36e4-460d-a98b-1b2d91f44b8c)
(I have to use another theme locally with respect to the ReadTheDocs site to compile properly).

It would be nice if you could (very) briefly describe the different filter methods (`WaveformsStdFilter` and `ChargeDistributionFilter`). Do you think it could be feasible?

jlenain commented on docs/user-guide/howto-pedestal.rst:
```diff
@@ -0,0 +1,36 @@
+
+The user script `nectarchain/user_scripts/ltibaldo/example_pedestal.py` showcases the usage of the tool.
+
+The input data are indentified by run number. See :ref:`env-vars` to set up the ``$NECTARCAMDATA`` environment variable. The pedestal estimation tool inherits the configurable parameters of the `~nectarchain.makers.component.PedestalComponent.PedestalEstimationComponent`.
```
Small typo: "indentified" -> identified

tibaldo replied:
Fixed.

tibaldo commented:
Hi @jlenain, thanks for the review. I implemented the addition you suggested. I also added some information about the number of events per slice and the bad pixel flagging.

jlenain commented:
Thank you very much, @tibaldo !
